tests/e2e/upgrade_test.go

```diff
@@ -37,17 +37,19 @@ var _ = Describe("E2E - Upgrading Elemental Operator", Label("upgrade-operator")
 	}
 
 	It("Upgrade operator", func() {
-		err := kubectl.RunHelmBinaryWithCustomErr("upgrade", "--install", "elemental-operator",
-			operatorUpgrade,
-			"--namespace", "cattle-elemental-system",
-			"--create-namespace",
-		)
-		Expect(err).To(Not(HaveOccurred()))
+		for _, chart := range []string{"elemental-operator-crds", "elemental-operator"} {
```

Comment:
This is the correct approach generally speaking and this is what we should stick with in the future. However this is what is likely to fail when upgrading from current stable to current dev. As the order should be inverted (just for the upgrade from one chart to any other release based on two charts).

Probably here the trick could be similar to the one before, if there is a previous crds chart installation use `[]string{"elemental-operator-crds", "elemental-operator"}` if there is no previous crds chart installation lets use `[]string{"elemental-operator", "elemental-operator-crds"}`.

Anyway I'd leave it for a follow up PR.